src/xrGame/player_hud.cpp

```diff
@@ -53,16 +53,20 @@ void player_hud_motion_container::load(IKinematicsAnimated* model, const shared_
             {
                 pm.m_base_name = anm;
                 pm.m_additional_name = anm;
+                pm.m_anim_speed = 1.f;
             }
             else
             {
-                R_ASSERT2(_GetItemCount(anm.c_str()) == 2, anm.c_str());
+                R_ASSERT2(_GetItemCount(anm.c_str()) <= 3, anm.c_str());
```

Comment:
The code below requires at least one parameter, and uses additional two.
It's more logical to use this condition:
```suggestion
                R_ASSERT2(_GetItemCount(anm.c_str()) > 0, anm.c_str());
```

Reply:
Logics before:
1 parameter - use name as base and additional parameter
2+ parameters - assert it is only 2 params and read both

Logics after:
1 parameter - same as before
2+ parameters - make sure it is either 2 or 3 parameters, if someone leaves empty space and wants to only add animation speed

---

Seems like:
1 - adding < 0 assertion and storing count as variant makes sense
2 - probably I should add trimming for pm.m_additional_name strlen check? Or parser of comma separated values already does it? Will need to check it